docs-ref-services/mysql.md

```diff
@@ -31,23 +31,22 @@ npm install mysql
 
 ## Example
 
-Connect to, and query, a MySQL database.
+Connect to a MySQL database and perform a simple query.
 
 ```javascript
-var mysql      = require('mysql');
-var connection = mysql.createConnection({
-  host     : 'localhost',
-  user     : 'me',
-  password : 'secret',
-  database : 'my_db'
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'me',
+  password: 'secret',
+  database: 'my_db'
 });
 
 connection.connect();
-
-connection.query('SELECT 1 + 1 AS solution', function (error, results, fields) {
-  if (error) throw error;
-  console.log('The solution is: ', results[0].solution);
-});
+const query = 'SELECT 1 + 1 AS solution';
```

Comment:
I like that this runs without any tables, but I think it is OK here to assume the reader will be more comfortable with a basic query like

`SELECT * FROM customers`

If you prefer, add some text to the above sentence that says 

> Connect to a MySQL database and perform a simple query to get all customers